Adyen/Service/Checkout.cs

```diff
@@ -106,11 +106,11 @@ public async Task<PaymentMethodsResponse> PaymentMethodsAsync(PaymentMethodsRequ
         /// </summary>
         /// <param name="paymentsDetailsRequest"></param>
         /// <returns>PaymentsResponse</returns>
-        public PaymentResponse PaymentDetails(PaymentsDetailsRequest paymentsDetailsRequest, RequestOptions requestOptions = null)
+        public PaymentDetailsResponse PaymentDetails(PaymentsDetailsRequest paymentsDetailsRequest, RequestOptions requestOptions = null)
         {
             var jsonRequest = Util.JsonOperation.SerializeRequest(paymentsDetailsRequest);
             var jsonResponse = _paymentDetails.Request(jsonRequest, requestOptions);
-            return Util.JsonOperation.Deserialize<PaymentResponse>(jsonResponse);
+            return Util.JsonOperation.Deserialize<PaymentDetailsResponse>(jsonResponse);
         }
 
         /// <summary>
```

Comment:
should the `PaymentDetailsAsync` method return a  `Task<PaymentDetailsResponse>` instead?